src/scancode/api.py

```diff
@@ -323,5 +323,5 @@ def extract_archives(location, recurse=True):
     """
     from extractcode.extract import extract
     from extractcode import default_kinds
-    for xevent in extract(location, kinds=default_kinds, recurse=recurse):
+    for xevent in extract(location, kinds=default_kinds, recurse=recurse, remove_originals=remove_originals):
         yield xevent
```

Comment:
Could it be that an alternative design would be to use the extract events location and target field (once the regular extraction took place) to actually to delete location and move/rename target in its place at once?

Reply:
Sounds like a better idea. I will look into that! Thank you!